e2e-tests/tfm/cypress/support/trade-finance-manager-api/getJwtTokenForUser.js

```diff
@@ -0,0 +1,6 @@
+const { getJwtTokenForUser } = require('./api');
+
+module.exports = (dealId, dealType, opts) => {
+  const { user } = opts;
+  return getJwtTokenForUser(dealId, dealType, null, user);
+};
```

Comment:
Documentation

Also perhaps `getJwtTokenForUser` should have object structured params, then we don't need to pass a `null` value

Reply:
This is old cypress command, API call was already removed before, removing this file now.